Gamecube/GamecubeMain.cpp

```diff
@@ -342,10 +342,26 @@ void video_mode_init(GXRModeObj *videomode, u32 *fb1, u32 *fb2, u32 *fb3)
 	xfb[2] = fb3;
 }
 
+bool Autoboot;
+char AutobootROM[1024];
+char AutobootPath[1024];
+
 int main(int argc, char *argv[]) 
 {
 	/* INITIALIZE */
 #ifdef HW_RVL
+	if(argc > 2 && argv[1] != NULL && argv[2] != NULL)
+	{
+		Autoboot = true;
+		strncpy(AutobootPath, argv[1], sizeof(AutobootPath));
+		strncpy(AutobootROM, argv[2], sizeof(AutobootROM));
+	}
+	else
+	{
+		Autoboot = false;
+		memset(AutobootPath, 0, sizeof(AutobootPath));
+		memset(AutobootROM, 0, sizeof(AutobootROM));
```

Comment:
These are the default, the whole "else" case can be dropped.

Reply:
i'm not sure about deleting the whole else.
We need also to tell the emulator if we are not autobooting games, then empty the autoboot variables and say Autoboot = false.

Reply:
If you are not sure, then maybe it's time you learn C++.